css/style.css

```diff
@@ -1,19 +1,196 @@
+    width:100%;
+    display: flex;
+    flex-direction: column;
+    margin:0 auto;
+}
+
+/* *******HEADER******* */
+.header{
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+} 
+.logo {
+    height: 80px;
+    width: 80px;
+    padding-left: 5rem;
+    margin: 2.5rem  0;
+}
+.nav_bar{
+    margin-top: 50px;
+}
+
+.navigation_list{
+    display:flex;
+    list-style: none;
+    padding-right: 5rem;  
+}
+
+.navigation_item{
+    padding:20px;
+    
+}
+
+.navigation_link {
+    color:rgb(109, 106, 106);
+    font-weight: 550;
+    font-size: 23px;
+    text-transform: uppercase;
+    text-decoration: none;
+
+}.navigation_link:hover{
+    color:rgb(245, 122, 21);
+}
+#meet_karma{
+    color:black;
+}
+#meet_karma:hover{
+    color:rgb(245, 122, 21);
+}
+
+/** MAIN***/
+
+.intro_karma{
+    width: 100%;
+    height: 100vh;
+    background-image: url("../img/first-background.jpg");  
+    background-size: cover; 
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    color:white;
+    margin-bottom: 50px;
+}
+
+.need_karma{
+    padding: 50px;
+}
 
+/*************TEXT STYLE*********/
+.intro_karma h1{
+    font-weight: 100;
+    font-size:70px;
+    margin-top:15rem;
+    
+}    
+.intro_karma h2{
+    font-weight: 50;
+    font-size: 40px;
+    margin-bottom:50px;
+}
+.need_karma h3{
+    text-align:center;
+    font-size: 50px;
+    font-weight: 40;
+    color: rgb(47, 45, 45);
+}
+.need_karma p{
+    color: rgb(47, 45, 45);
+    font-size:25px;
+
+}
 
+/********BUTTON********/
+button{
+    background-color:rgb(185, 82, 45);
+    color:white;
+    border-radius: 0.3rem;
+    padding: 15px 5px;
+    width:10rem;
+}
+/******Features*****/
+.features{
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    margin-bottom: 50px;
+}
+.feature_icon{
+     display: flex;
+     flex-direction: column;
+     align-items: center;
+     font-size: 1.5rem;
+     text-align: center;
+     padding:50px;
+     margin: 0 20px;     
+}
+.icon{
+    width: 10rem;
+    height: 10rem;
+}
+/****Footer*****/
+.footer {
+    display: flex;
+    flex-direction: column; 
+    align-items: center;  
+} 
+
+
+.find_us{
+    width:25px;
+    height: 25px; 
+    padding:15px;
+    border: 3px solid rgb(184, 180, 180);
+    border-radius:45%;
+}
+hr{
+    width:80%;
+    color:whitesmoke;
+}
+
+/**IMG  &  QUOTE */
+ .img_quote_container{
+     display: flex;
+     flex-flow: row wrap;
+     width:100%;
+     margin-bottom: 100px;
+     
+ }
+ .img_of_girl{
+     width:45%;
+
+ }
+ .quote{
+     width:55%;
+     padding-top: 10rem;
+     text-align: center;
+     background-color: rgb(245, 229, 232);
+ }
+ .quote p{
+     font-size: 35px;
+     font-weight: 45;
+ }
+ q{
+     display:inline;
+ }
+ q::before{
+     content: open-quote;
+ }
+ q::after{
+     content: close-quote;
+ }
+ .standing_quortation{
+
+        font-style: italic;
+        font-size: 56px;
+        color:#ce7508;
+        
+ }
```

Comment:
I think there is a spelling mistake in the .standing_quortation class reference (.standing_quotation). Also, the spacing of the properties in this class could be reduced in order to be aligned with the spacing of the other properties.